cypress/fixtures/example.json

```diff
@@ -0,0 +1,5 @@
+{
```

Comment:
Skal vi fjerne dette, siden vi ikke bruker det? Kan heller legge det til når vi har bruk for det :)

Reply:
Fjernet :)